app/service/send_pdf_letter_schema.py

```diff
@@ -8,6 +8,7 @@
         "filename": {"type": "string"},
         "created_by": {"type": "string"},
         "file_id": {"type": "string"},
+        "recipient_address": {"type": "string"}
     },
-    "required": ["postage", "filename", "created_by", "file_id"]
+    "required": ["postage", "filename", "created_by", "file_id", "recipient_address"]
```

Comment:
If you try and upload a letter with no address you now get a `500` error page instead of being able to send the letter. Maybe we need to add a story to make template preview check that there is actually an address at some point so that the error message is clearer

Reply:
Good thinking - I'll add a story.

Reply:
This has been done. The template-preview app will return a validation error if the address is missing.